crates/hotshot/examples/infra/mod.rs

```diff
@@ -842,12 +842,16 @@ pub async fn main_entry_point<
     let node_index = run_config.node_index;
     error!("Retrieved config; our node index is {node_index}");
 
+    // We use input seed to calculate node's own key pair because its type is too complex to directly input through the config file
     run_config.config.my_own_validator_config =
         ValidatorConfig::<<TYPES as NodeType>::SignatureKey>::generated_from_seed_indexed(
             run_config.seed,
             node_index,
             1,
         );
+    // Here is a way to generate peer's public key, in practice we should input peer's stake_value and public key
+    run_config.config.known_nodes_with_stake =
```

Comment:
In practice, now that nodes generate their own keys, how will nodes fetch peer public keys + stake? Maybe an example would be helpful

Reply:
@nomaxg  As mentioned in the above `Comments requested`, the second point, peer's configuration (including peer's public key and stake value) is my biggest concern. Now they are all share the same seed and generated at the entry which is exactly where you pointed out. Once I asked @jbearer during testnet meeting about this, he mentioned it's fine. But would be better to double check whether we want to use any interface in `hotshot` to get peers' configuration. Although I suppose the sequencer is performing all updates and HotShot is taking it as an input.

Reply:
Yes I believe we will need some method of fetching peers' configuration (public key/stake?/etc.), if sequencer nodes are generating their own keys for Cappuccino